### PR TITLE
Fix for keeping int values

### PR DESCRIPTION
**Pull request by victorv-sbp**

### CODE REVIEW

**Comment by joakimnordling** [comment]
@victorv-sbp: Could you provide a minimal example of the issue?

**Comment by lietu** [comment]
Definitely needs at minimum a unit test demonstrating the problem and that this fixes it

File: devops/lib/utils.py [hunk 1]
```diff
@@ -183,7 +183,10 @@ def _nest(_doc, _overrides, _path):
                     res[key] = doc[key]
                 elif type(overrides[key]) in (str, int, bool, float, complex):
                     # Simply overridden values
-                    res[key] = _basevalue_to_value(overrides[key], path)
+                    if type(_basevalue_to_value(overrides[key], path)) == int:
```
Comment (soderluk) [inline devops/lib/utils.py]: What about boolean values? such as `allowPrivilegeEscalation: true` which is converted to `allowPrivilegeEscalation: 'True'`. This also fails validation.

**Comment by lietu** [comment]
The whole point of `_basevalue_to_value` is to convert those string values to int/bool/float/complex etc. - if you have identified bugs with that function, fix them inside that function

**Comment by lietu** [comment]
This has been solved by #23

**Pull request closed without merging**